src/scss/colors.scss

```diff
@@ -21,7 +21,7 @@ $alphas-enum: (0.18, 0.2, 0.24, 0.35, 0.5, 0.56, 0.65, 0.8);
 
     @each $alpha in $alphas-enum {
       --#{$name}-accent--under-#{$alpha*100}a: #{rgba(
-          var(--primary-accent--under-rbg),
+          var(--accent--under-rbg),
           $alpha
         )};
```

Comment:
`$colors-map` since it is in the form key: value.
No `accent` segment in the name since it comes from `$colors-map`.

```scss
 @each $name, $color in $colors-enum {
   @each $alpha in $alphas-enum {
      --#{$name}-#{$alpha*100}a: #{rgba(
          var(--#{$name}),
          $alpha
        )};
    }
}
```

Reply:
the one thing - `$color-map` contain HEX format of #7c46fb and here `rgba` requires only RBG format

Reply:
K, this `var(--#{$name}-rgb)`  instead `var(--#{$name})`  👈  doesn't invalidate the aforementioned 2 items:

- $colors-map since it is in the form key: value.
- No accent segment in the name since it comes from $colors-map.

Reply:
make sense, thank for a help
fixed in 0cd2600f5d7e357895b206741557c96ce99cd816